lib/cocoapods/target/aggregate_target.rb

```diff
@@ -184,6 +184,69 @@ def uses_swift?
       pod_targets.any?(&:uses_swift?)
     end
 
+    # @return [Hash{ Hash{ Symbol => String}}] The vendored dynamic artifacts and framework targets grouped by config
+    #
+    def frameworks_by_config
```

Comment:
I think this method might need a bit of a cleanup and perhaps split into two methods.

One that returns the frameworks per config and the other that decorates the frameworks with all the paths